readme.md

```diff
@@ -16,6 +16,11 @@ The Terrarium project aims to empower platform engineering professionals by prov
 
 ## Installation Instructions
 
+### Prerequisite
+
+<img alt="Terraform" src="https://www.datocms-assets.com/2885/1629941242-logo-terraform-main.svg" width="150px">                     <img alt="Golang" src="https://raw.githubusercontent.com/github/explore/80688e429a7d4ef2fca1e82350fe8e3517d3494d/topics/yaml/yaml.png" width="40px">                    <img alt="Terraform" src="https://user-images.githubusercontent.com/25181517/192149581-88194d20-1a37-4be8-8801-5dc0017ffbbe.png" width="50px">
```

Comment:
The Terraform logo text is very unreadable on dark background. Just the square icon may be better.

Reply:
thanks for the feedback peter,
i will check and change it according to both bright and dark background